modules/web/js/ballerina/components/compound-statement-decorator.jsx

```diff
@@ -53,6 +53,13 @@ class CompoundStatementDecorator extends React.Component {
         this.setState({innerDropZoneExist: false});
     }
 
+    onJumptoCodeLine() {
```

Comment:
If this method is an overridden method, we need to set {@inheritDoc} here and the super class should have the actual comment. Unless the behavior is changed.